vars/javaLibs.groovy

```diff
@@ -39,6 +39,10 @@ def call(Map configMap = [:], Closure stepsConfigCls) {
       stage('Preparation') {
         agent any
         steps {
+          script {
+            env.RELEASE_TYPE = params.RELEASE_TYPE?: "snapshot"
+            env.RELEASE_SCOPE = params.RELEASE_SCOPE?: ""
```

Comment:
I would wish for a more formulized way of setting the default. Could we at least define a const value per file with the defaults so we use this in the params block and here?

Reply:
Done.